sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/resourcemanager/AzureResourceManagerAutoConfigurationTest.java

```diff
@@ -60,4 +65,44 @@ void testWithoutAzureResourceMetadataClass() {
                           .run(context -> assertThat(context).doesNotHaveBean(AzureProfile.class));
     }
 
+    @Test
+    void testAzureProfileWithAzureDefault() {
+        this.contextRunner
+            .withUserConfiguration(AzureGlobalPropertiesAutoConfiguration.class)
+            .withBean(AzureResourceManager.class, AzureResourceManagerExt::getAzureResourceManager)
+            .withPropertyValues(
+                "spring.cloud.azure.profile.tenant-id=test-tenant-id",
+                "spring.cloud.azure.profile.subscription-id=test-subscription-id"
+            )
+            .run(context -> {
+                assertThat(context).hasSingleBean(AzureProfile.class);
+                AzureProfile azureProfile = context.getBean(AzureProfile.class);
+                Assertions.assertEquals(azureProfile.getEnvironment().getActiveDirectoryEndpoint(),
+                    AZURE.getActiveDirectoryEndpoint());
+            });
+    }
+
+    @Test
+    void testAzureProfileWithAzureChina() {
+        this.contextRunner
+            .withUserConfiguration(AzureGlobalPropertiesAutoConfiguration.class)
+            .withBean(AzureResourceManager.class, AzureResourceManagerExt::getAzureResourceManager)
+            .withPropertyValues(
+                "spring.cloud.azure.profile.tenant-id=test-tenant-id",
+                "spring.cloud.azure.profile.subscription-id=test-subscription-id",
+                "spring.cloud.azure.profile.cloud=AZURE_CHINA"
+            )
+            .run(context -> {
+                assertThat(context).hasSingleBean(AzureProfile.class);
+                AzureProfile azureProfile = context.getBean(AzureProfile.class);
+                Assertions.assertEquals(azureProfile.getEnvironment().getActiveDirectoryEndpoint(),
```

Comment:
I assume these two properties are not relevant to resource manager?

Reply:
You mean the `tenant-id` and `subscription-id`? it's required for AzureResourceManagerAutoConfiguration, https://github.com/Azure/azure-sdk-for-java/blob/15dda6cdc3219e9128a4d5207cb66d891fd1baf8/sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/resourcemanager/AzureResourceManagerAutoConfiguration.java#L25